.appveyor.yml

```diff
@@ -11,14 +11,15 @@ build: off
 deploy: off
 
 cache:
-  - node_modules
-  - "%LOCALAPPDATA%/Yarn"
+  - node_modules -> appveyor.yml, package.json, yarn.lock
+  - "%LOCALAPPDATA%/Yarn -> appveyor.yml, package.json, yarn.lock"
```

Comment:
Setup cache invalidation :+1: to avoid problem in future

Reply:
nice!